### DISCUSSION TITLE
Documentation Miss

### DISCUSSION THREAD

**Discussion by karmaniverous** [mmistakes/minimal-mistakes#3774] [Show and tell]
Hey @mmistakes OMFG you have transformed the Jekyll experience for me. THANK YOU!!!

FYI I ran into trouble when I cloned the starter template and then tried to run locally. The fix was easy but not obvious, detailed [here](https://blog.karmanivero.us/blog/hello-again-world/) if you'd like to update your docs.

**Comment by iBug** [comment]
For the record, Webrick used to come included with Jekyll, which changed in either Jekyll 4.2.0 or Ruby 3.0 (or both).

**Reply by karmaniverous** [reply]
Lol figures. Anyway thanks again for the amazing theme!
